kyo-core/native/src/main/scala/kyo/addersStubs.scala

```diff
@@ -0,0 +1,29 @@
+package java.util.concurrent.atomic
+
+class DoubleAdder():
```

Comment:
I've created these quick stubs but we should follow up later optimizing them since `synchronized` isn't ideal.